play-interop-test-scala/src/test/scala/akka/grpc/gen/PlayScalaRouterSpec.scala

```diff
@@ -29,7 +30,7 @@ class PlayScalaRouterSpec extends WordSpec with Matchers with BeforeAndAfterAll
   implicit val ec = sys.dispatcher
   implicit val patience = PatienceConfig(timeout = 3.seconds, interval = 15.milliseconds)
 
-  val router = new GreeterServiceImpl
+  val router = new GreeterServiceRouter(new GreeterServiceImpl)
```

Comment:
Here you can see how a new Scala router and service are composed together.

Reply:
Ah, this looks nice than `val router = new GreeterServiceImpl`! 👍🏼